src/odb/src/db/dbAccessPoint.h

```diff
@@ -75,6 +76,9 @@ class _dbAccessPoint : public _dbObject
   dbAccessType::Value high_type_;
   dbVector<dbId<_dbITerm>>
       iterms_;  // list of iterms that prefer this access point
+  dbVector<dbVector<std::pair<dbObjectType, dbId<_dbObject>>>>
+      vias_;  // list of vias by num of cuts
+  dbVector<Rect> path_segs_;
```

Comment:
@Mahfouz-z move inside the user code section

Reply:
Code produced via generator now.